### ISSUE TITLE
Incorrectly reports dependency error during install

### ISSUE THREAD

**Issue by rajinder-yadav** [palantir/tslint#1626]
### Bug Report

- __TSLint version__: 3.15.1
- __TypeScript version__: 2.0.3

I am using nvm and getting the error reported below.

```
~$ npm install -g tslint
/Users/yadav/.nvm/versions/node/v6.7.0/bin/tslint -> /Users/yadav/.nvm/versions/node/v6.7.0/lib/node_modules/tslint/bin/tslint
/Users/yadav/.nvm/versions/node/v6.7.0/lib
├── tslint@3.15.1
└── UNMET PEER DEPENDENCY typescript@>=1.7.3

npm WARN tslint@3.15.1 requires a peer of typescript@>=1.7.3 but none was installed.
```

**Comment by adidahiya** [comment]
duplicate of #815
